agent/wptdriver/wpt_settings.cc

```diff
@@ -102,8 +103,20 @@ bool WptSettings::Load(void) {
   lstrcpy( PathFindFileName(iniFile), _T("wptdriver.ini") );
   _ini_file = iniFile;
   lstrcpy(logFile, iniFile);
-  lstrcpy( PathFindFileName(logFile), _T("wpt.log") );
+
+  // setting logs
+  lstrcpy(PathFindFileName(logFile), _T("wpt.log"));
   DeleteFile(logFile);
+  global_logfile_handle = CreateFile(logFile, GENERIC_WRITE, FILE_SHARE_WRITE | FILE_SHARE_READ,
+    NULL, OPEN_ALWAYS, 0, 0);
+  if (logfile_handle == INVALID_HANDLE_VALUE) {
+    WptTrace(loglevel::kFunction, _T("Failed to open log file. Error: %d"), GetLastError());

```

Comment:
Would be nice to output a string with the error as well. I believe FormatMessage gives you that.